drift/example/main.g.dart

```diff
@@ -939,7 +939,7 @@ final class $$TodoItemsTableReferences
       db.todoCategories.createAlias(
           $_aliasNameGenerator(db.todoItems.categoryId, db.todoCategories.id));
 
-  $$TodoCategoriesTableProcessedTableManager? get categoryId {
+  $$TodoCategoriesTableProcessedTableManager get categoryId {
     if ($_item.categoryId == null) return null;
```

Comment:
That `return null` doesn't look correct to me here, shouldn't we do a null assertion (`!`) instead? Or does this just need to be re-generated (the `drift_dev` changes seem to suggest that, and the `examples/app` code doesn't have this either).